src/components/landing/BuildingSection.svelte

```diff
@@ -1,5 +1,11 @@
 <script>
   import { InfoSplitSection, DottedAccent } from '../'
+  import missionContent from '@contentful-entries/missionSection'
+  const content = missionContent.find(
+    mission => mission.key === 'building-section'
+  )
```

Comment:
`import content from '@contentful-entry/missionSection/building-section'`

Reply:
Wait, we took out the `key` logic from the `rollup-plugin-content-loader.js`! Recommend you add it back in.

Reply:
Just revert [this part](https://github.com/GTBitsOfGood/bog-web/commit/10ce85bbd356d108a22e5db0128429871c472fd4#diff-1c14294a91a89fc5e48f7e39672a75d1L107-L113).